src/olympia/translations/models.py

```diff
@@ -251,11 +250,13 @@ def __str__(self):
         return str(self.localized_string_clean)
 
     def clean(self):
-        # URL_RE is the regexp used by bleach to detect URLs to linkify them,
-        # in our case we use it to find them and replace them with nothing.
-        # It's more effective/aggressive than something like r'http\S+', it can
-        # also detect things like foo.com.
-        self.localized_string_clean = URL_RE.sub('', self.localized_string).strip()
+        from olympia.amo.utils import URL_RE
+
+        self.localized_string_clean = (
+            URL_RE.sub('', self.localized_string).strip()
+            if self.localized_string
+            else self.localized_string
+        )
```

Comment:
```suggestion
        self.localized_string_clean = URL_RE.sub('', self.localized_string or '').strip()
```?

Reply:
That would transform `None` into an empty string which I'm not sure we want to do at this point.